src/components/Button/styles/styles.module.scss

```diff
@@ -0,0 +1,19 @@
+@import "../../../assests/variables.module.scss";
+
+.confirmButton {
+  padding: 5px 10px;
+  border-width: 0px;
+  border-radius: 3px;
+  background-color: $tertiary-color;
+  color: $background-secondary-color;
+  font-size: 1em;
+}
+
+.primaryButton {
+  padding: 5px 10px;
+  border-width: 0px;
+  border-radius: 3px;
+  background-color: $primary-color;
+  color: $background-secondary-color;
+  font-size: 1em;
+}
```

Comment:
1. If you are repeating a code, in most cases you are doing something wrong. Plus, consider you have `sass`. You can write this styles a way better.
2. Please define variables for spacing and radius and fonts.

Reply:
donnnnnnnnnnneeeeeee